### PR TITLE
Language android13

### PR DESCRIPTION
**Pull request by criticalAY**
for Android 13 compat: Use new language support #991

### CODE REVIEW

File: app/src/main/java/protect/card_locker/AboutActivity.java [hunk 1]
```diff
@@ -62,6 +62,7 @@ protected void onCreate(Bundle savedInstanceState) {
         }
 
         final List<ThirdPartyInfo> USED_LIBRARIES = new ArrayList<>();
+        USED_LIBRARIES.add(new ThirdPartyInfo("uCrop","https://github.com/Yalantis/uCrop.","Apache 2.0"));
```
Comment (Altonss) [inline app/src/main/java/protect/card_locker/AboutActivity.java:65]: Why this appears here, isn't that already on  the master branch? :thinking:

Comment (criticalAY) [reply app/src/main/java/protect/card_locker/AboutActivity.java]: fixed

**Comment by criticalAY** [comment]
Ah! I guess my code was updated on my system and I did not notice it sorry

**Comment by Altonss** [comment]
> Ah! I guess my code was updated on my system and I did not notice it sorry

Hmm now it seems the whole file has been deleted...

**Comment by criticalAY** [comment]
> > Ah! I guess my code was updated on my system and I did not notice it sorry
> 
> Hmm now it seems the whole file has been deleted...

how to fix it now? could you please help?

**Comment by Altonss** [comment]
> > > Ah! I guess my code was updated on my system and I did not notice it sorry
> > 
> > 
> > Hmm now it seems the whole file has been deleted...
> 
> how to fix it now? could you please help?

Well normally your changes to the language part should just be applied on top of the current master branch... I'm not a git expert, but maybe you can just cherry-pick the right commits containing only the language content and apply them on top of the master branch

**Comment by criticalAY** [comment]
> > > > Ah! I guess my code was updated on my system and I did not notice it sorry
> > > 
> > > 
> > > Hmm now it seems the whole file has been deleted...
> > 
> > 
> > how to fix it now? could you please help?
> 
> Well normally your changes to the language part should just be applied on top of the current master branch... I'm not a git expert, but maybe you can just cherry-pick the right commits containing only the language content and apply them on top of the master branch

so do i need to make a new PR now?

**Comment by Altonss** [comment]
> so do i need to make a new PR now?

I don't know :thinking: maybe @TheLastProject has an idea?

**Comment by criticalAY** [comment]
> > so do i need to make a new PR now?
> 
> I don't know 🤔 maybe @TheLastProject has an idea?

can we make a communication channel on Discord or any other platform i.e. Matrix? It would be easier to communicate then and issue like these could be handled there only.

**Comment by TheLastProject** [comment]
Well, this is more a git issue than a Catima issue.

I always prevent these things from happening by for every new feature first going back to the master branch and git pulling it, then making a new branch and only then starting to do work.

For now, given it's only a single file, the easiest fix is probably to just download the file from GitHub and commit that.

**Comment by TheLastProject** [comment]
> can we make a communication channel on Discord or any other platform i.e. Matrix? It would be easier to communicate then and issue like these could be handled there only.

Not opposed to a Matrix chat but also, Catima is already taking a lot of time from me and people on chat platforms tend to expect an answer within a minute always and I really don't have the time to always respond to people like that.

**Comment by TheLastProject** [comment]
Superseded by #1033.

**Pull request closed without merging**